app/src/main/java/com/medmapper/v33001/MainActivity.kt

```diff
@@ -29,8 +29,7 @@ class MainActivity : ComponentActivity() {
     }
 }
 
-@Composable
-fun Greeting(name: String) {
+val Greeting: @Composable (String) -> Unit = { name ->
```

Comment:
Changed this to a Lambda - this looks neater
Probably should have added a default value for first time use, and then replaced with the name every time afterwards

Reply:
I kinda like it the way it was.  Putting all of that on one line is a lot of symbols... and the @Composable annotation is hidden, where it is usually on top of a Composable function.

Putting too much on one line can make debugging tricky.